src/commands.rs

```diff
@@ -33,7 +33,7 @@ pub enum Commands {
         /// If omitted, mdBook uses build.build-dir from book.toml \
         /// or defaults to `./book`.
         #[clap(long, short, value_hint = ValueHint::DirPath)]
-        dest_dir: Option<PathBuf>,
```

Comment:
This breaks with our idea of backward compatibility

https://rust-lang.github.io/mdBook/cli/build.html#--dest-dir